.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,55 @@
+- [ ] Add [CHANGELOG](https://github.com/timescale/timescaledb/blob/master/CHANGELOG.md) updates
+- [ ] Needs [documentation](https://github.com/timescale/docs) updates
+
+Fixes #<issue number>.
+
+---
+Cut everything below the above line.
+
+## Guidelines for Pull requests
+
+Checklist for a pull request:
+
+- Try to follow [the seven rules of a great Git commit
+message](https://chris.beams.io/posts/git-commit/).
+- Ideally, the PR should be merged as a single commit.
+- Rebase on latest master code.
+- Reference the issue(s) resolved with Fixes #<issue number>, or
+  Closes #<issue number>.
```

Comment:
Should we standardize on one label?

Reply:
IMO, it is not important. I also think they can serve slightly different purposes. For example, "Fixes" make more sense to use for bug issues, while Closes applies more broadly to, e.g., features and performance tweaks that aren't fixes.